src/CalcViewModel/DataLoaders/UnitConverterDataLoader.cpp

```diff
@@ -156,27 +156,32 @@ void UnitConverterDataLoader::GetCategories(_In_ shared_ptr<vector<UCM::Category
 
 void UnitConverterDataLoader::GetUnits(_In_ unordered_map<ViewMode, vector<OrderedUnit>>& unitMap)
 {
-    bool USSource, USTarget;
-    bool UKSource, UKTarget;
-    bool Source, Target;
+    // US + Federated States of Micronesia, Marshall Islands, Palau
+    bool isPartOfCOFA = m_currentRegionCode == L"US" ||
+        m_currentRegionCode == L"PW" ||
+        m_currentRegionCode == L"FM" ||
+        m_currentRegionCode == L"MH" ||
+        m_currentRegionCode == L"PW";
 
-    USSource = (GetRegion() == L"US") ? true : false;
-    USTarget = USSource;
+    // COPA + Liberia
+    // Source: https://en.wikipedia.org/wiki/Metrication
+    bool useUSCustomary = isPartOfCOFA || m_currentRegionCode == L"LR";
+    bool useISU = !useUSCustomary;
```

Comment:
nit: I don't think "ISU" is a commonly-used abbreviation. Consider "useSI" or "useMetric"

Reply:
Note that I did my best to not use a French acronym!

Reply:
SI (~ System International) is known in math/science even in the US...